core/src/main/resources/lib/hudson/iconSize_pt_BR.properties

```
@@ -21,6 +21,6 @@
 # THE SOFTWARE.
 
 Icon=Ícone
-L=L
-S=S
+S=P
+L=G
 M=M
```

Note that [the English versions of these](https://github.com/jenkinsci/jenkins/blob/34d098dfaa5097e77b4de621a4d0af236432d2a5/core/src/main/resources/lib/hudson/iconSize.properties#L24-L26) are "Small", "Medium", and "Large", and [the rendering code](https://github.com/jenkinsci/jenkins/blob/34d098dfaa5097e77b4de621a4d0af236432d2a5/core/src/main/resources/lib/hudson/iconSize.jelly) displays the first letter on the page while displaying the full word as a tooltip. In its current form, this PR only displays the first letter on the page and as a tooltip, which is inconsistent with the English version. To be consistent, this should include the full word:

```suggestion
S=Pequeno
L=Grande
M=Médio
```